controller.py

```diff
@@ -1,24 +1,33 @@
 import os
 import time
-from caus import CAUS
-from kubernetes import client, config
-from elasticity import elasticity
-from prometheusclient import prometheusMonitor 
-DEPLOYMENT_NAME = "nginx-deployment"
+from caus import CAUS, SimpleCAUS
+from config import get_config
+from kubernetes import client, config as kubernetes_config
+from elasticity import Elasticity
+from prometheusclient import PrometheusMonitor
+from configparser import ConfigParser
+
+config: ConfigParser = get_config()
+
+DEPLOYMENT_NAME: str = config['deployment'].get('deployment-name', 'nginx-deployment')
 
 # Creates an object from a file with the given path or creates default object -> TODO
-def create_deployment_object_from_file(fullpath: str=""):
+def create_deployment(fullpath: str = "") -> client.V1Deployment :
```

Comment:
I'm still a bit confused about this method  
to me, this looks like a specific deployment for testing purposes 
Or at least I don't see a reason why a general CAUS would need an nginx  
Aren't there existing config files which can be used to create deployments? The old name also suggest support for sth like this

Reply:
… Until you look at https://github.com/t2-project/caus/blob/cce276d221b4d144f13f7d3402f5a795d7a7d92f/controller.py#L53-L55 and see that it has never been implemented

Reply:
Also, I don't know why it needs an nginx either…

Reply:
yes, I saw that 
my issue with this is mainly, if it is out of scope, then removing it completely would probably be better  
or implement it properly  
but the current state feels hacky

Reply:
Good, then I'll remove it.

Reply:
you resolved this comment, but if I see correctly, the code is still there  
is this on purpose?

Reply:
https://github.com/t2-project/caus/blob/7a12887bb7b0cd5ddfa34b2c1d285946fc163901/controller.py#L15-L59

Reply:
We are talking about the same thing, right?
The fact that the `fullpath` param was unused, right?

Reply:
If you had the assumption because you clicked on the link of the comment above: That is because GitHub redirects only to the specific commit the comment was mentioned on.
While it does make somewhat sense, it is also at least equally idiotic.

Reply:
well, for me also the whole `create_deployment` method  
at least to me it looks like a dev test method  
as we both did not find a reason why CAUS in general would need an nginx

Reply:
I have a slight guess for that:
To me, it looks like you need any deployment for the code below to work correctly.
Probably the simplest thing that came to mind was to misuse an nginx deployment as a NOP-deployment…

Reply:
well, I guess that's obvious  
CAUS is an autoscaler, and an autoscaler needs a deployment to scale  
the remaining question for me, does a simple dummy deployment have to be part of the api of CAUS? I would say no. A Typical workflow would be to create your deployment manually or with any tool you like, and then throw CAUS on that deployment to scale it.  
Having a function to create a deployment from a what so ever config file seems to make somewhat sense to me, however this was considered to be out of scope. But a method to create a simple dummy deployment seems more like a dev debug tool